service-deployment/bootstrap/prometheus/prometheus-configmap.yaml.template

```diff
@@ -23,18 +23,22 @@ data:
   prometheus.yml: |-
     # Scrape config for cluster components.
     scrape_configs: 
-    - job_name: "node_exporter"
-      scrape_interval: "15s"
```

Comment:
>scrape_interval: "15s" [](start = 6, length = 22)

if removed, where do you specify the scrape interval?

Reply:
if removed, will use default value.

Reply:
we need to control the scrape interval. please fix it.

---
In reply to: [176297106](https://github.com/Microsoft/pai/pull/395#discussion_r176297106) [](ancestors = 176297106)

Reply:
added